src/5e-SRD-Magic-Items.json

```diff
@@ -244,6 +244,90 @@
       "If the bag is pierced or torn, it is destroyed, and anything contained within it is transported to a random location on the Astral Plane."
     ],
     "url": "/api/magic-items/bag-of-devouring"
+  },
+   {
+    "index": "bag-of-holding",
```

Comment:
You should also add these into the `5e-SRD-Equipment-Categories.json` as well. Look at my PR for more details.

Reply:
got it

Reply:
done!